torch/distributed/elastic/agent/server/__init__.py

```diff
@@ -29,6 +29,20 @@
 in the same job) to make a collective decision.
 """
 
+__all__ = [
+    # local elastic agent
+    "TORCHELASTIC_ENABLE_FILE_TIMER",
+    "TORCHELASTIC_TIMER_FILE",
+    # api
+    "ElasticAgent",
+    "RunResult",
+    "SimpleElasticAgent",
+    "Worker",
+    "WorkerGroup",
+    "WorkerSpec",
+    "WorkerState",
+]
+
 from .api import (  # noqa: F401
```

Comment:
Since we have a `__all__` export list now, we are able to remove the `# noqa: F401` pragma.